django/cantusdb_project/main_app/templates/source_detail.html

```diff
@@ -175,7 +175,10 @@ <h4>{{ source.siglum }}</h4>
                         </select>
 
                         <br>
-                        <a href="{% url "chant-list" %}?source={{ source.id }}" class="guillemet" target="_blank">View all chants</a>
+                        {% if source.segment.id == 4063 %}
```

Comment:
I assume that 4063 here is the auto-assigned ID of the CantusDB segment in the database?

Reply:
It's the same ID used on OldCantus afaik, so we manually assigned it. https://github.com/DDMAL/CantusDB/wiki/Segment-Model